File: 6.Software Technologies/Exam_Prep/Babies/C#/SoftUniBabies/Models/Baby.cs
﻿namespace SoftUniBabies.Models
{
    using System;
    using System.ComponentModel.DataAnnotations;

    public class Baby
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Gender { get; set; }
        public string Birthday { get; set; }

    }
}

File: 6.Software Technologies/Exam_Prep/Babies/JavaScript/models/baby.js
const Sequelize = require('sequelize');

module.exports = function (sequelize) {
    const Baby = sequelize.define('Baby', {
        name: {
            type:Sequelize.TEXT,
            required:true,
            allowNull:false
        },
        gender: {
            type:Sequelize.TEXT,
            required:true,
            allowNull:false
        },
        birthday:{
            type:Sequelize.TEXT,
            required:true,
            allowNull:false
        }
    },{
        timestamps:false
    });

    return Baby;
};